javascript/javascript1/week2/formalname.js

```diff
@@ -0,0 +1,47 @@
+let fullName;
+let firstName;
+let surName;
+let useFormalName;
+
+function getFullname(firstName, surName, useFormalName) {
```

Comment:
Nice parameter naming!
If you want, you can make it even more clear by making the boolean value a  yes/no-question:
`function getFullname(firstName, surName, shouldUseFormalName) {`